dot/services.go

```diff
@@ -247,6 +248,11 @@ func createNetworkService(cfg *Config, stateSrvc *state.Service) (*network.Servi
 		cfg.Core.Roles, cfg.Network.Port, strings.Join(cfg.Network.Bootnodes, ","), cfg.Network.ProtocolID,
 		cfg.Network.NoBootstrap, cfg.Network.NoMDNS)
 
+	slotDuration, err := stateSrvc.Epoch.GetSlotDuration()
+	if err != nil {
+		return nil, err
```

Comment:
We should add some context to the error. i.e.

```suggestion
		return nil, fmt.Errorf("cannot get slot duration: %w", err)
```

Reply:
Done